package.json

```diff
@@ -548,6 +553,11 @@
                     "default": true,
                     "description": "If set to true, the extension will automatically watch and scan the network for online Roku devices. This can be pared with the ${promptForHost} option in the launch config to display a list of online Rokus, removing the need to constantly change the host IP in your config files.",
                     "scope": "resource"
+                },
+                "brightscript.rokuAdvancedLayoutEditor.trackerTaskFileLocation": {
```

Comment:
I understand wanting this to be a user setting, and that's fine to support it. However, we may want to consider supporting this as a launch config setting as well. The launch.json setting would override this value if found.

Reply:
The reason this is a user setting is because this is workspace independent. I'd be happy to hope on a short call to talk over this feature.